web/themes/custom/contribtracker/package.json

```diff
@@ -45,7 +45,13 @@
     "typescript": "^4.8.4"
   },
   "dependencies": {
-    "select2": "^4.1.0-rc.0"
+    "rimraf": "^5.0.5",
+    "select2": "^4.1.0-rc.0",
+    "uuid": "^9.0.1"
```

Comment:
@AlviMurtaza, why are these dependencies needed? Or did you mean to add them to `devDependencies`?

Reply:
We were getting a similar error after the updates. https://github.com/isaacs/rimraf/issues/264 Adding this dependency resolved the issue.

Reply:
Sure, but my question is if this should be a part of `devDependencies`?

Reply:
Updated